docs/kb/semgrep-ci/ci-vs-cli.md

```diff
@@ -44,9 +44,9 @@ would give similar results to `semgrep ci`.
 
 ## Pro analysis
 
-When using `semgrep ci` with Semgrep AppSec Platform, the Semgrep scan reflects the Pro Engine options configured in Semgrep AppSec Platform at https://semgrep.dev/orgs/-/settings. If Pro Engine is enabled, and the scanned code includes [supported Pro languages](/docs/supported-languages/#semgrep-code-language-support), then interfile and interprocedural analysis is performed.
+When using `semgrep ci` with the Semgrep AppSec Platform, the Semgrep scan reflects the Pro Engine capabilities available in Semgrep AppSec Platform at https://semgrep.dev/orgs/-/settings. The only configurable Pro Engine feature in the UI is Cross-File Analysis. Cross-File Analysis if enabled, Semgrep performs interfile and interprocedural analysis for supported Pro languages.
```

Comment:
```suggestion
When using `semgrep ci` with Semgrep AppSec Platform, you can configure whether the scan uses cross-file analysis in [Settings](https://semgrep.dev/orgs/-/settings). If you enable cross-file analysis, Semgrep performs interfile and interprocedural analysis for supported Pro languages.
```

Reply:
Do we still use the term "Pro languages", or would plain "for supported languages" be better here?

Reply:
Good call -- we can use supported languages and link to https://semgrep.dev/docs/supported-languages#language-maturity-summary (I can add the link)

Reply:
nice!